fire/cli/mtl.py

```diff
@@ -841,3 +908,356 @@ def regn(projektnavn: str, **kwargs) -> None:
+    except:
+        sagsgang = pd.DataFrame(
+            columns=["Dato", "Initialer", "Hændelse", "Tekst", "uuid"],
+            dtype={
+                "Dato": "datetime64[ns]",
+                "Initialer": "string",
+                "Hændelse": "string",
+                "Tekst": "string",
+                "uuid": "string",
+            },
+        )
+        assert sagsgang.shape[0] == 0, "Forventede tom dataframe"
+    return sagsgang
+
+
+# ------------------------------------------------------------------------------
+def find_sagsid(sagsgang: pd.DataFrame) -> str:
+    sag = sagsgang.index[sagsgang["Hændelse"] == "sagsoprettelse"].tolist()
+    assert (
+        len(sag) == 1
+    ), "Der skal være præcis 1 hændelse af type sagsoprettelse i arket"
+    i = sag[0]
+    if False == pd.isna(sagsgang.uuid[i]):
+        return str(sagsgang.uuid[i])
+    return ""
+
+
+def find_alle_landsnumre_i_distrikt(distrikt: str) -> List[str]:
+    pit = firedb.hent_punktinformationtype("IDENT:landsnr")
+    landsnumre = (
+        firedb.session.query(PunktInformation)
+        .filter(
+            PunktInformation.infotypeid == pit.infotypeid,
+            PunktInformation.tekst.startswith(distrikt),
+        )
+        .all()
+    )
+    løbenumre = [n.tekst.split("-")[-1] for n in landsnumre if "-" in n.tekst]
+    numre = [int(n) for n in løbenumre if n.isnumeric()]
+    return set(numre)
+
+
+# ------------------------------------------------------------------------------
+# Her starter punktregistreringsprogrammet...
+# ------------------------------------------------------------------------------
+@mtl.command()
+@fire.cli.default_options()
+@click.argument(
+    "projektnavn", nargs=1, type=str,
+)
+@click.argument(
+    "initialer", nargs=1, type=str,
+)
+def registrer_punkter(projektnavn: str, initialer: str, **kwargs) -> None:
+    """Registrer nyoprettede punkter i databasen"""
+    check_om_resultatregneark_er_lukket(projektnavn)
+    fire.cli.print("Så registrerer vi")
+    resultater = {}
+
+    sagsgang = find_sagsgang(projektnavn)
+    sagsid = find_sagsid(sagsgang)
+    try:
+        sag = firedb.hent_sag(sagsid)
+    except:
+        fire.cli.print(
+            f" Sag for {projektnavn} er endnu ikke oprettet - brug fire mtl opret-sag! ",
+            bold=True,
+            bg="red",
+        )
+    if not sag.aktiv:
+        fire.cli.print(
+            f"Sag {sagsid} for {projektnavn} er markeret inaktiv. Genåbn for at gå videre."
+        )
+        return
+
+    # -----------------------------------------------------
+    # Opbyg oversigt over nyetablerede punkter
+    # -----------------------------------------------------
+    nyetablerede = find_nyetablerede(projektnavn)
+    nye_punkter = set(nyetablerede.index)
+    nyetablerede = nyetablerede.reset_index()
+    n = nyetablerede.shape[0]
+
+    if n == 0:
+        fire.cli.print("Ingen nyetablerede punkter at registrere")
+        return
+
+    v = Koordinatvrider()
```

Comment:
`v` er ikke et særligt godt navn her. Jeg er typisk tilhænger af at navngive variablen efter klassen (så længe der kun er brug for en udgave af den):

```
koordinatvrider = Koordinatvrider()
```

men se min kommentar længere oppe angående klassens (og tilhørende metodes) navn